client/lib/signup/page-builder.js

```diff
@@ -21,6 +21,11 @@ export function shouldEnterPageBuilder() {
 	return inTest;
 }
 
+export function getEditHomeUrl( siteSlug ) {
+	// @todo we will need to retrieve the home page ID from site options
+	return `/block-editor/page/${ siteSlug }/2`;
```

Comment:
Will you get this from the backend?

If not, if you didn't already know, there are some selectors on the frontend that might assist, e.g., `getSiteFrontPage` and `getGutenbergEditorUrl`

https://github.com/Automattic/wp-calypso/blob/3acf6fb/client/my-sites/checklist/wpcom-checklist/component.jsx#L986

https://github.com/Automattic/wp-calypso/blob/3acf6fb/client/state/selectors/get-gutenberg-editor-url.js

Reply:
Thanks for linking those! I was aware of `getSiteFrontPage` but not `getGutenbergEditorUrl`